aiida/cmdline/params/types/group.py

```diff
@@ -82,3 +86,64 @@ def convert(self, value, param, ctx):
                 raise
 
         return group
+
+
+class GroupPartialParamType(LabelStringType):
```

Comment:
Don't think this actually works as is, does it? You define the `orm_class_loader` property, but that is not actually used by the `LabelStringType` class. Did you mean to subclass from the `GroupParamType`? That subclasses the `IdentifierParamType` that actually knows how to load the instance from a value for the given entity class.

Reply:
Ah, yeah, I forgot to remove it. I was using it for the autocomplete since I originally was doing the same thing as `GroupParamType.shell_complete` but had to change it to be able to reproduce the same behavior as the list command. BTW I tried describing this in the docstrings of the class and the method but not sure how clear it is, it would be good to have a second pair of eyes check it.

But other than that the functionality works for me locally, I just defined the `orm_class_loader` unnecessarily.